.github/workflows/cypress-tests-runner.yml

```diff
@@ -151,6 +151,11 @@ jobs:
           DATABASE_URL: postgres://db_user:db_pass@localhost:5432/hyperswitch_db
         run: just migrate run --locked-schema
 
+      - name: Insert card info into the database
+        if: ${{ env.RUN_TESTS == 'true' }}
+        run: |
+          PGPASSWORD=db_pass psql -h localhost -p 5432 -U db_user -d hyperswitch_db --command "\copy cards_info FROM '.github/data/cards_info.csv' DELIMITER ',' CSV HEADER;"
```

Comment:
Nit: Can we use the longer variant of these flags, `--host` instead of `-h`, etc.?